trident-get-started/requirements.adoc

```diff
@@ -100,7 +101,7 @@ By default, Astra Trident runs in a container and will, therefore, run on any Li
 
 Though Astra Trident does not officially "support" specific operating systems, the following Linux distributions are known to work:
 
-* RedHat CoreOS 4.2 and 4.3
+* RedHat CoreOS (RHCOS) versions as supported by OpenShift Container Platform
```

Comment:
Also check how I worded this statement. Basically we support the RHCOS versions that are included with OpenShift versions we support.